src/checkpoint/container/view_serialize.h

```diff
@@ -424,6 +424,14 @@ void serializeExtentOnly(SerializerT& s, Kokkos::View<T**,Ts...>& v, std::string
   }
 }
 
+template< typename SerializerT, typename T, typename... Ts >
+void serializeContentsOnly(SerializerT& s, Kokkos::View<T, Ts...>& v) {
```

Comment:
Great addition! 
With this addition, what should be the meaning of `s | v` ? 
I wonder whether the user will expect to get the contents of v serialized (at least, I was expecting it and Matt B. could not explain the 0 entries in the output view).

Reply:
Your issue is likely that two different objects both have serialization code that reads s|v on View instances that refer to a common underlying array. When deserialized, each of them gets back a fresh instance. Those serialization routines need to be coordinated, such that one object owns the view and takes responsibility for serializing it, and the others are coordinated to obtain the same reference.

The function I add here will have no impact on plain s|v serialization of a few - it only applies where explicitly called. 

In the use case of EMPIRE restoration, where the entire scaffolding of a job will be rebuilt from the input deck, it may be appropriate to pretty much uniformly call serializeContentsOnly or nothing at all on View members, trusting that the initialization has guaranteed that all objects have correctly aliased referenced to any Views of shared arrays.

Reply:
Sounds good. Thank you for clarifying.